api/grpc/types/api.proto

```diff
@@ -98,6 +98,7 @@ message User {
 }
 
 message AddProcessResponse {
+	uint32 systemPid = 1;
```

Comment:
No preferences here, the struct is already called `AddProcessResponse`, so it's explicit enough for me. `SystemPid` works :)